lib/sportradar/api/nfl/season.rb

```diff
@@ -27,6 +29,9 @@ def set_weeks
         end
       end
 
+      # set_(conferences|divisions|teams) are all identical to the same methods in Hierarchy
+      # There is likely more overlap between Sportradar's NFL standings and NFL hierarchy APIs
```

Comment:
Agreed, that's for later on. I should have been more clear on that, these comments are mostly just to help cross-reference later.